tests/bulk/bulkwrite-insert-003.phpt

```diff
@@ -11,30 +11,25 @@ require_once __DIR__ . "/../utils/basic.inc";
 $manager = create_test_manager();
 
 $document = MongoDB\BSON\Document::fromPHP(['foo' => 'bar']);
-$array = MongoDB\BSON\PackedArray::fromPHP(['bar']);
 
 $bulk = new MongoDB\Driver\BulkWrite();
 
-$insertedDocumentId = $bulk->insert($document);
-$insertedArrayId = $bulk->insert($array);
```

Comment:
The PackedArray insertion was entirely removed from this test since it's no longer allowed.